README.md

```diff
@@ -357,53 +357,61 @@ the "run" command. E.g., continuing from the example above, it can be:
-representation. The `stdio` reporter doesn't need any parameters, so you can
-just pass an empty object, like this:
+#### mail Reporter
 
-    skt --rc skt-rc --state --workdir skt-workdir -vv \
-        report --reporter stdio '{}'
+The `mail` reporter sends an email to one or more recipients with the results
+of the testing. Multiple options are available:
 
-The `mail` reporter parameters are a bit more involved and to include the
-"From" address, and a list of "To" addresses for the message to send:
+* `--mail-to`: one or more recipient email addresses *(required)*
+* `--mail-from`: the email address of the sender *(required)*
+* `--mail-subject`: the email subject *(optional)*
+* `--mail-header`: one or more email headers to add to the email *(optional)*
 
-    skt --rc <SKTRC> --state --workdir <WORKDIR> -vv \
-        report --reporter mail \
-        '{"mailfrom": "<MAILFROM>", "mailto": "<MAILTO_LIST>"}'
+The most basic email report can be sent using these arguments:
 
-Here, `<MAILFROM>` would be the "From" address, and `<MAILTO_LIST>`
-would be a comma-separated list of "To" addresses.
+    skt --rc skt-rc --state --workdir skt-workdir -vv \
+        report --reporter mail \
+        --mail-to developer@example.com \
+        --mail-from skt@example.com
 
-The following example sends the report to the current user and to root on the
-same host, with "From" address being the current user:
+Multiple recipient addresses can be provided:
 
     skt --rc skt-rc --state --workdir skt-workdir -vv \
         report --reporter mail \
-        '{"mailfrom": "'$USER'@localhost", "mailto": "'$USER'@localhost, root@localhost"}'
+        --mail-to developer@example.com --mail-to dev2@example.com \
+        --mail-from skt@example.com
 
-Note that the `report` command will reach for build artifacts via the URLs
-generated by the `publish` command.
+Here is a full example with multiple email headers:
 
-`reporter` command is able to send a single report for multiple test runs. For
-now, only runs with same patch series and base are supported. This is
-especially useful if the same patch series are being tested on multiple
-architectures. To run reporter in multireport mode, add `--states` option
+    skt --rc skt-rc --state --workdir skt-workdir -vv \
+        report --reporter mail \
+        --mail-to developer@example.com dev2@example.com \
+        --mail-from skt@example.com \
+        --mail-subject "Important tests results" \
+        --mail-header "X-Build-ID: 225" \
+        --mail-header "In-Reply-To: <messageid@example.com>"
+
+The `reporter` command is able to send a single report for multiple test
+runs. For now, only runs with same patch series and base are supported. This
+is especially useful if the same patch series are being tested on multiple
+architectures. To run reporter in multireport mode, add `--result` option
 with a list of skt state files created by `run` command you wish to report.
```

Comment:
This sentence should be edited more, to say "... add `--result` options with skt state files...` (or something similar) since it no longer accepts lists.

Reply:
Fixed!